vagrant/src/main/vagrant/servers.yaml

```diff
@@ -56,23 +56,43 @@ servers:
         - sudo systemctl start brooklyn
         - sudo systemctl enable brooklyn
   - name: byon1
-    box: ubuntu/wily64
+    box: centos/7
     ram: 512
     cpus: 2
     ip: 10.10.10.101
+    shell:
+      env: {}
+      cmd:
+        - sudo sed -i 's/PasswordAuthentication no/PasswordAuthentication yes/g' /etc/ssh/sshd_config
+        - sudo service sshd restart
   - name: byon2
-    box: ubuntu/wily64
+    box: centos/7
```

Comment:
Worth going with xenial or cents for all boxes, it avoids the user having to download 2 distros? If I recall correctly there had been issues standing up Tomcat on the trusty vagrant box hence the use of Wily - worth testing the getting started blueprints on the new distros.

Also re testing at one point I'd looked at the feasibility of CI for the vagrant getting started and it looked doable using Software to request on metal boxes (so you run virtual box)